app/src/utils/data/contributions.ts

```diff
@@ -20,30 +20,29 @@ const { grantRoundManager } = useWalletStore();
  * @notice Get/Refresh all contributions
  *
  * @param {number} blockNumber The current blockNumber
- * @param {Object} grantsDict A dict of grant addresses (grant.id->grant.payee)
+ * @param {Object} grantPayees A dict of grant addresses (grant.id->grant.payee)
  * @param {TokenInfo} donationToken The roundManagers donation token
  * @param {boolean} forceRefresh Force the cache to refresh
  */
 export async function getContributions(
   blockNumber: number,
-  grantsDict: Record<string, string>,
+  grantPayees: Record<string, string>,
   donationToken: TokenInfo,
   forceRefresh?: boolean
 ) {
   return await syncStorage(
     contributionsKey,
     {
       blockNumber: blockNumber,
-      timeStamp: Date.now() / 1000,
     },
-    async (localStorageData?: LocalStorageAnyObj | undefined, save?: (saveData?: LocalStorageAnyObj) => void) => {
-      const rawData = localStorageData?.data || {};
-      const contributions: Record<string, Contribution> = rawData?.contributions || {};
+    async (LocalForageData?: LocalForageAnyObj | undefined, save?: (saveData?: LocalForageAnyObj) => void) => {
+      // pick up contributions from the localStorage obj
+      const ls_contributions: Record<string, Contribution> = LocalForageData?.data?.contributions || {};
```

Comment:
Similar comment to above on preferring `lsContributions` vs `ls_contributions`

Reply:
I've updated to follow the convention you've set in `cart.ts` (for localStorage versions of data) 👍